utils/ipfs/ipfs_test.go

```diff
@@ -4,21 +4,33 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"encoding/hex"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
 	"github.com/machinefi/sprout/utils/ipfs"
 )
 
-func DISABLE_TestIPFS(t *testing.T) {
-	endpoint := "ipfs.mainnet.iotex.io"
+func TestIPFS(t *testing.T) {
 	r := require.New(t)
+	s := ipfs.NewIPFS("ipfs.mainnet.iotex.io")
 
 	content := []byte("test data")
+	f, err := os.OpenFile("test", os.O_CREATE|os.O_RDWR, 0666)
```

Comment:
use os.CreateTemp?
https://pkg.go.dev/os#CreateTemp

Reply:
The absolute path of the file is used. cannot use `os.CreateTemp`